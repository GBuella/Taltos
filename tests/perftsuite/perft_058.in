setboard 6kq/8/8/8/8/8/8/7K w - - 0 1
echo 6kq/8/8/8/8/8/8/7K w - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
