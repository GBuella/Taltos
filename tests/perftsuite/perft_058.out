6kq/8/8/8/8/8/8/7K w - - 0 1
2
36
143
3637
14893
391507
